code/modules/clothing/spacesuits/rig/modules/ds13/kinesis/extension.dm

```diff
@@ -32,7 +32,7 @@
 	ripple = subject.filters[subject.filters.len]
 	animate(ripple, radius = 16, size = 1, time = 4, loop = -1, flags = ANIMATION_PARALLEL)
 
-	outline = filter(type = "outline", size = 3, color = COLOR_KINESIS_INDIGO, alpha = 128)
```

Comment:
this is a functional change, revert

Reply:
filter type outline don't support alpha. Check DM references filter outline

Reply:
It does, I've tested